src/controllers/reportArticle.js

```diff
@@ -0,0 +1,61 @@
+import models from '../db/models';
+import errorResponse from '../helpers';
+
+const { Report } = models;
+
+/**
+ * @description A controller class for handling comment business logic
+ *
+ * @class Comment
+ */
+class ReportArticle {
+  /**
+   * @description - reports an article
+   *
+   * @static
+   * @param {object} request - request object
+   * @param {object} response - response object
+   *
+   * @returns {object} - error or success response
+   *
+   * @memberof Comment
+   */
+  static async report(request, response) {
```

Comment:
Nice work with this controller, I see you're not making any room for repetitions and you're using your variables efficiently 👍🏿